packages/peregrine/lib/talons/CartPage/GiftCards/useGiftCard.js

```diff
@@ -11,3 +25,12 @@ export const useGiftCard = props => {
         removeGiftCardWithCode
     };
 };
+
+/** JSDoc type definitions */
+
+/**
+ * Props data to use when rendering a single gift card component.
+ * @typedef {Object} GiftCardProps
```

Comment:
This might be nitpicky but these aren't _exactly_ `GiftCard` component props. A component may need more than what its talon returns.

We've conflated the terms by calling return values from talons `talonProps` throughout the code, so maybe for that consistency this should stay.

Reply:
Yeah, that could work 👍 . I'm just trying to differentiate the talon's return value from the props that the `GiftCard` component (in this case) needs or expects.

Reply:
Resolved in 350ef11